app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerActivityTest.kt

```diff
@@ -170,7 +175,7 @@ class NavigationDrawerActivityTest {
   }
 
   @Test
-  fun testNavigationDrawerTestActivity_openNavDrawer_changeConfig_navDrawerIsDisplayed() {
+  fun testNavDrawer_openNavDrawer_changeConfig_navDrawerIsDisplayed() {
```

Comment:
let's keep it `configChange` everywhere, as this is what we are following in other PRs as well.

Reply:
Done